Development.xcconfig

```diff
@@ -1 +1 @@
-OTHER_SWIFT_FLAGS=$(inherited) -Xfrontend -warn-long-expression-type-checking=400
+OTHER_SWIFT_FLAGS=$(inherited) -DSWIFT_PACKAGE -Xfrontend -warn-long-expression-type-checking=400
```

Comment:
This may not be necessary, but was a temporary fix for swift-web tests:

https://github.com/pointfreeco/swift-web/compare/pointfreeco:bf4e7f8...pointfreeco:3292555